packages/react/src/transactions/adapters/profiles/BlockProfilesGateway.ts

```diff
@@ -66,11 +71,10 @@ export class BlockProfilesGateway
       id: result.id,
       request,
       typedData: omitTypename(result.typedData),
-      fallback: this.createRequestFallback(request, result),
     });
   }
 
-  private async relayWithProfileManager(
+  private async broadcast(
```

Comment:
Intentional?

Reply:
intentional, but only to keep consistency between gateways - i see they've been renamed to this now.

will revert back